src/containers/genres/js/genres.js

```diff
@@ -22,25 +23,31 @@ function uncheck(event) {
 function storeGenreOptions(event) {
     event.preventDefault();
 
-    var yesGenreArray = [];
-    var noGenreArray = [];
+    var checked = document.querySelectorAll("input:checked")
 
-    for (var i = 0; i < inputYes.length; i++) {
-        if (inputYes[i].checked === true) {
-            yesGenreArray.push(inputYes[i].dataset.index);
-        };
-    };
+    if (checked.length === 0) {
+        alert("Please select at least one genre")
```

Comment:
Here the message is going to be displayed, but it seems that the execution will continue anyway, so you need to exit the function with a return.